cmd/vic-machine/create/create.go

```diff
@@ -379,8 +379,37 @@ func (c *Create) processContainerNetworks() error {
 		c.MappedNetworksIPRanges[vicnet] = pools[vnet]
 		c.MappedNetworksDNS[vicnet] = dns[vnet]
 
+		delete(gws, vnet)
+		delete(pools, vnet)
+		delete(dns, vnet)
 	}
 
+	var hasError bool
+	if len(gws) > 0 {
+		log.Errorf("Following container network gateway is set, but CONTAINER-NETWORK not found. Please check your --container-network and --container-network-gateway")
+		for key, value := range gws {
+			mask, _ := value.Mask.Size()
+			log.Errorf("\t%s:%s/%d, %q should be vsphere network name", key, value.IP, mask, key)
+		}
+		hasError = true
+	}
+	if len(pools) > 0 {
+		log.Errorf("Following container network ip range is set, but CONTAINER-NETWORK not found. Please check your --container-network and --container-network-ip-range")
```

Comment:
Should be: "The following container network IP range is set, but CONTAINER-NETWORK cannot be not found. Please check the --container-network and --container-network-ip-range settings"